panther_battery/src/adc_node.py

```diff
@@ -221,10 +221,25 @@ def _publish_battery_msg(
             if self._charger_connected:
                 if battery_msg.percentage == 1.0:
                     battery_msg.power_supply_status = BatteryState.POWER_SUPPLY_STATUS_FULL
+                    rospy.loginfo_throttle(
+                        60.0,
+                        f'[{rospy.get_name()}] The battery is fully charged. '
+                        f'The robot can be disconnected from the charger.',
+                    )
                 elif I_bat_mean > self._I_bat_charging_thresh[bat_pub]:
                     battery_msg.power_supply_status = BatteryState.POWER_SUPPLY_STATUS_CHARGING
+                    rospy.loginfo_throttle(
+                        1800.0,
+                        f'[{rospy.get_name()}] Robot charging process update: \n',
+                        f'    - Battery Percentage: {battery_msg.percentage * 100}%',
+                    )
```

Comment:
```suggestion
                    rospy.loginfo_throttle(
                        1800.0,
                        f'[{rospy.get_name()}] Robot charging process update. '
                        f'Battery Percentage: {battery_msg.percentage * 100}%',
                    )
```
As I think about right now, there is no need to log this as a list, one-liner should look good. Also, comma after the first part of the string will make the code fail.